plugins/fetcher.cc

```diff
@@ -25,9 +28,15 @@ namespace MLDB {
 /** Function that fetches a row from a url. */
 
 struct FetcherFunctionConfig {
-    FetcherFunctionConfig()
+    FetcherFunctionConfig() : maxConcurrentFetch(-1),
```

Comment:
Do these inline and remove the constructor, eg int maxConcurrentFetch = -1.  Also, there shouldn't be a mutex or a condition variable in the config object; that belongs in the fetcher itself.